src/components/fields/DescriptionField.js

```diff
@@ -10,17 +10,17 @@ function DescriptionField(props) {
     return <div />;
   }
 
-  const renderers = {};
+  const renderers = window.renderers || {};
```

Comment:
What's with putting the `renderers` on `window`? I don't understand why this is being done; are you trying to find a way to pass some object from one React element to another but can't find a way to do it properly?

Reply:
We might not need to do this at all as I have suggested an alternate way to handle markdown rendering in another comment.